src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AndroidBrokerProxy.cs

```diff
@@ -286,12 +301,12 @@ private Account FindAccount(string accountName, Account[] accountList)
             {
                 foreach (Account account in accountList)
                 {
-                    bool found = account != null && 
-                                 account.Name != null && 
+                    bool found = account != null &&
+                                 account.Name != null &&
```

Comment:
if account.Name is a string, you should use string.IsNullOrEmpty instead of a null comparison